src/structures/api/Participant.ts

```diff
@@ -588,6 +596,9 @@ export class Participant {
       mitigated: data.damageSelfMitigated
     };
 
+    this.totalAllyJungleMinionsKilled = data.totalAllyJungleMinionsKilled;
+    this.totalEnemyJungleMinionsKilled = data.totalEnemyJungleMinionsKilled;
```

Comment:
Could this be used to fine-tune the `creepScore` property? Still never know what CS numbers are right.

https://github.com/TheDrone7/shieldbow/blob/42eb3c1d4fc678dcad7acfbc9f926d4e2d2a450a/src/structures/api/Participant.ts#L654